sphinx/util/__init__.py

```diff
@@ -172,6 +174,9 @@ def copy_static_entry(source, targetdir, builder, context={},
 
     Handles all possible cases of files, directories and subdirectories.
     """
+    warnings.warn('sphinx.util.copy_static_entry is deprecated for removal',
+                  RemovedInSphinx20Warning)
```

Comment:
See our deprecation policy. It should be deprecated in 3.0.
https://github.com/sphinx-doc/sphinx/blob/master/CONTRIBUTING.rst#deprecating-a-feature

I know this is not a feature, but we should also wait a moment on removal.

Reply:
But both of these have been marked as deprecated for years now. I'm just making it more obvious with warnings. I don't think there's any reason not to do this.

Reply:
In my experience, nobody reads docs and CHANGES. So it would be better to emit deprecation warnings.
If extensions won't work after upgrade, users will be in trouble. So I'd like to emit deprecation warnings obviously to provide a deprecation period for users and developers.

Reply:
OK, let me rework these so.

Reply:
Thank you for update!
Sorry for boring fix. These deprecation with only comments were my mistakes.